content/organization/events/online/index.en.md

```diff
@@ -88,52 +88,52 @@ At the very bottom you should see some buttons -- click on the 'Claim Host' butt
 
 {{< figure src="enter-key.png" alt="Screenshot of the 'enter key' form in the Zoom interface" >}}
 
-### Tips for Running the Zoom meeting
+### Tips for running the Zoom meeting
 
-:eyes: After becoming Host, we suggest that you:
+The guidance in this section comes from the Baltimore, Irvine, NYC, and Santa Barbara chapter guidelines.
 
--   Enable the waiting room.
+:eyes: After becoming the Host, we suggest that you:
 
--   Admit your co-organisers (if they are still in the waiting room) and
- make them co-host so that they can manage bad actors with you if
- necessary. **Being several to run an online meeting makes things easier (e.g. one organizer can keep an eye on the chat).** 
+-   Admit your co-organizers from the waiting room and
+ make them co-hosts so that they can manage bad actors with you if
+ necessary. **Having multiple co-organizers run an online meeting makes things much easier (e.g., one organizer can keep an eye on the chat).** 
  It also makes it possible for your co-host to claim host if you for some reason disconnect or experience other technical difficulties.
-We recommend being at minimum one co-host to help you.
+We recommend having (at a minimum) one co-host to help you. Three to four co-hosts in total would be ideal.
```

Comment:
This might not be possible for all chapters. How about 
```suggestion
We recommend having (at a minimum) one co-host to help you. For larger events, you may consider three to four co-hosts.
```